tests/macros/utils.js

```diff
@@ -1,86 +1,243 @@
+    expect(x).toBeTruthy();
 }
 
+/**
+ * Asserts non-strict equality (==) of actual and expected.
+ *
+ * @template T Type of the objects.
+ * @param {T} x Actual value.
+ * @param {any} y Potential expected value.
+ */
 assert.equal = (x, y) => {
     expect(x).toEqual(y);
 };
 
 assert.eventually = {
+    /**
+     * Asserts non-strict equality (==) of actual and expected.
+     *
+     * @template T Type of the objects.
+     * @param {T|PromiseLike<T>} x Actual value.
+     * @param {any} y Potential expected value.
+     */
     async equal(x, y) {
         expect(await x).toEqual(y);
     }
 };
 
-assert.include = (list, element) => {
-    expect(list).toContain(element);
-};
+/**
+ * Asserts that value is true.
+ *
+ * @template T Type of value.
+ * @param {T} value Actual value.
+ */
 assert.isTrue = value => {
     expect(value).toEqual(true);
 };
+
+/**
+ * Asserts that value is false.
+ *
+ * @template T Type of value.
+ * @param {T} value Actual value.
+ */
 assert.isFalse = value => {
     expect(value).toEqual(false);
 };
+
+/**
+ * Asserts value is strictly greater than (>) floor.
+ *
+ * @param {number} value Actual value.
+ * @param {number} floor Minimum Potential expected value.
+ */
 assert.isAbove = (value, floor) => {
     expect(value).toBeGreaterThan(floor);
 };
+
+/**
+ * Asserts that value is an array.
+ *
+ * @template T Type of value.
+ * @param {T} value Actual value.
+ */
 assert.isArray = value => {
     expect(value).toBeInstanceOf(Array);
 };
+
+/**
+ * Asserts that value is an object of type 'Object'
+ * (as revealed by Object.prototype.toString).
+ *
+ * @template T Type of value.
+ * @param {T} value Actual value.
+ * @remarks The assertion does match subclassed objects.
+ */
 assert.isObject = value => {
     expect(value).toBeInstanceOf(Object);
 };
+
+/**
+ * Asserts that value is a function.
+ *
+ * @template T Type of value.
+ * @param {T} value Actual value.
+ */
 assert.isFunction = value => {
     expect(value).toBeInstanceOf(Function);
 };
+
+/**
+ * Asserts that object has a property named by property.
+ *
+ * @template T Type of object.
+ * @param {T} value Container object.
+ * @param {string} prop Potential contained property of object.
+ */
 assert.property = (value, prop) => {
     expect(value).toHaveProperty(prop);
 };
+
+/**
+ * Asserts that object doesn't have a property named by property.
+ *
+ * @template T Type of object.
+ * @param {T} value Container object.
+ * @param {string} prop Potential contained property of object.
+ */
 assert.notProperty = (value, prop) => {
     expect(value).not.toHaveProperty(prop);
 };
+
+/**
+ * Asserts that set1 and set2 have the same members. Order is not take into account.
+ *
+ * @template T Type of set values.
+ * @param {T[]} a1 Actual set of values.
+ * @param {T[]} a2 Potential expected set of values.
+ */
 assert.sameMembers = (a1, a2) => {
     expect(new Set(a1)).toEqual(new Set(a2));
 };
 
+/**
+ * Asserts that haystack includes needle.
```

Comment:
You use "haystack" and "needle" in some places and "list" and "element" in others.

Reply:
nit: it would be a slightly more minimal diff if this function didn't move within the list of assertion functions.